src/components/FlawComments.vue

```diff
@@ -30,12 +30,18 @@ function parseGroups(serializedJson: string) {
   }
 }
 
+function parseIsPrivate(isPrivate: string) {
```

Comment:
Wouldnt isParsePrivate be a better name?

Reply:
The purpose is this function is to parse the field `is_private`

Reply:
If we had an individual component (`FlawComment` singular) for each comment and passed in that `comment.meta-attr` data object as a prop, I would probably use a computed property instead of this function and call it `isPrivate`